_posts/2013-04-11-testing.md

```diff
@@ -60,20 +60,20 @@ import { test } from 'ember-qunit';
 import startApp from '../helpers/start-app';
 var App;
 
-module('An Integration test', {
-  setup: function() {
+Qunit.module('An Integration test', {
```

Comment:
ah, ok. Fine for now, but going forward we should shim qunit so we can `import { module } from 'qunit';`